introduction.md

```diff
@@ -0,0 +1,2 @@
+hi, I used vi add introduction.md for this course
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.